FV3/atmos_cubed_sphere/driver/fvGFS/atmosphere.F90

```diff
@@ -245,7 +244,8 @@ module atmosphere_mod
   integer :: isd, ied, jsd, jed
   integer :: nq                       !  number of transported tracers
   integer :: sec, seconds, days
-  integer :: id_dynam, id_fv_diag, id_subgridz
+  integer :: id_dynam = -1, id_subgridz = -1, id_dynam_other = -1
+  integer :: id_update = -1, id_fv_diag = -1, id_update_other = -1
```

Comment:
why do these need to be initialized to -1? no objection.

Reply:
Safety. If they happen to be used with an unitialized value that corresponds to an existing timer, we might get erroneous timings.